doc/user_guide/install.rst

```diff
@@ -319,6 +318,30 @@ From the root folder of your hyperspy repository (folder containing the
 
     $ pip install -e . --no-deps # install the currently checked-out branch of hyperspy
 
+Note that some tests only pass on specific packages versions (see :ref:`Plot testing<plot-test-label>`).
+Consequently, a previous version of python (such as 3.10) can end up being 
+installed in the ``hspy_dev`` environment. If this is a problem, ``hyperspy-dev`` should be 
+removed from environment creation. This will ensure that the latest python release is used. 
+
+Skipping ``hyperspy-dev`` will not prevent a successful installation from source, 
+but will require manually installing some packages. Notably, ``gui-ipywidgets`` 
+and ``gui-traitsui`` will be missing. More generally, all dependencies from ``dev`` 
+will be absent (see :ref:`install-with-pip`). These can be installed manually 
+using conda (inspect ``project.optional-dependencies`` section of 
+`pyproject.toml <https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/pyproject.toml>`__ 
+and install them using ``conda install -c conda-forge``). Alternatively, all 
+optional dependencies installation can be left to ``pip``:
+
+.. code-block:: bash
+
+    $ conda create -n hspy_dev python # create an env with latest stable python
+    $ conda activate hspy_dev # activate environment
+    $ conda install hyperspy-base -c conda-forge --only-deps # install hyperspy dependencies
+    $ pip install -e .[dev] # Install hyperspy from source + dev dependencies (run from source folder)
+
+Because this will result in many conflict risks being cleared by pip, this is not 
+necessarily the cleanest way to proceed.
```

Comment:
?

Reply:
The original install from source within a conda environment was failing (due to matplotlib not being forward-compatible with python 3.11+).  So I ended up modifying the guide to make it work again, but then you're stuck  with  python 3.10. 

This paragraph explain how to install from source on python 3.11+, both with and without pip for dependencies, and that there's a risk of failing the image tests